src/components/Button/button.module.scss

```diff
@@ -9,10 +9,10 @@
     --button-focus-outline-color: var(--primary-color-80);
     --button-active-outline-color: var(--primary-color-80);
     --button-visited-outline-color: var(--primary-color-80);
-    --button-primary-default-background-color: var(--primary-color);
-    --button-primary-hover-background-color: var(--primary-color-60);
+    --button-primary-default-background-color: var(--accent-color-20);
```

Comment:
Shouldn't this replace --primary-color? There are an equal number of color palettes meant to replace all the current ones. Should be easier just to swap them out completely as this is theme 2 and none of the theme 1 colors will be relevant to this branch. https://www.figma.com/file/SlKRC7oKF7XZyHMv2op4ch/Octuple-DS-(Theme-2)?node-id=125%3A3